_posts/2024-08-03-geospatial-techno.md

```diff
@@ -0,0 +1,120 @@
+---
+author: Milad Rafiei
+layout: post
+title: Using Binary Comparison Operators in GeoServer Filters
+date: 2024-08-03
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### Binary Comparison Operators in GeoServer Filters
+In this session, we want to talk about the various types of filters, with a particular focus on "Binary comparison operators in GeoServer" comprehensively. If you want to access the complete tutorial, simply click on the [link](https://www.youtube.com/watch?v=3XCwZIJxi7M&list=PL_ITaxp1Ob4sjk24Stboa5XbO0LGdEKbL).
```

Comment:
we try and leave out words like "simply" or "obviously" in technical writing as if someone is frustrated such words can make them feel stupid or not smart enough to use the technology.

Amusingly I found the YouTube showing me adds on a mobile device so annoying that it was not a "simple" experience to click on your link above!